product_docs/docs/eprs/7/installing/windows.mdx

```diff
@@ -33,19 +33,9 @@ In some versions of Windows, to invoke the installer with administrator privileg
 
 Proceed to [Using the graphical installer](#using-the-graphical-installer).
 
-## Using Stack Builder or StackBuilder Plus
+## Using StackBuilder Plus
 
-If you're using PostgreSQL, you can invoke the graphical installer with Stack Builder. See [Using Stack Builder](https://www.enterprisedb.com/docs/supported-open-source/postgresql/installing/03_using_stackbuilder/). 
```

Comment:
@anwar-sahib 
How can customers on PostgreSQL install EPRS on Windows if Stack Builder is no longer available? 
Can they use Stack Builder Plus? Or do they have to go for the "direct installation" path (installation wizard)?

Reply:
There is a way to directly install EPRS for Windows via website - https://www.enterprisedb.com/software-downloads-postgres#replication-server